funnel/templates/macros.html.jinja2

```diff
@@ -125,7 +125,7 @@
         <div class="project-banner__profile-details">
           {%- if project.profile.logo_url.url %}
             <a href="{{ project.profile.url_for() }}" class="project-banner__profile-details__logo-wrapper" data-action="View {{ project.profile.title }}(project page)">
-              <img class="project-banner__profile-details__logo_wrapper__logo" src="{{ project.profile.logo_url }}" alt="{{ project.profile.title }}"/>
+              <img class="project-banner__profile-details__logo_wrapper__logo" src="{{ project.profile.logo_url }}?size=80" alt="{{ project.profile.title }}"/>
```

Comment:
Jinja2 calls should use `{{ project.profile.logo_url.resize(80) }}` @iambibhas please confirm the method name and parameters.

This is required because the URL may use Imgee's S3 bucket instead of directly being an Imgee URL. The ImgeeField is smarter about knowing when it can resize and when it can't.

Reply:
Profile model was not using `ImgeeField`. `resize(width, [height])` is available to all `ImgeeField` instances.